great_expectations/data_context/store/configuration_store.py

```diff
@@ -157,7 +157,7 @@ def self_check(self, pretty_print: bool = True) -> dict:  # type: ignore[overrid
     def serialization_self_check(self, pretty_print: bool) -> None:
         raise NotImplementedError
 
-    def _determine_key(
+    def get_key(
```

Comment:
Rename to be consistent

Reply:
The hope is that all stores are consistent in their APIs eventually (we have a lot of wrapper convenience methods that create inconsistency and hurt polymorphism)